python/grass/pygrass/rpc/base.py

```diff
@@ -173,7 +173,9 @@ def safe_receive(self, message):
         except (EOFError, OSError, FatalError) as e:
             # The pipe was closed by the checker thread because
             # the server process was killed
-            raise FatalError("Exception raised: " + str(e) + " Message: " + message)
+            raise FatalError(
+                "Exception raised: " + str(e) + " Message: " + message
+            ) from e
```

Comment:
```suggestion
            ) from None
```

Reply:
This one I'm not sure we'd want this, because there are multiple types of errors caught. So the traceback will be able to tell us which one is the direct cause of the others